src/components/LoanCards/ListLoanCard.vue

```diff
@@ -1,13 +1,154 @@
 <template>
-	<div class="list-loan-card">
-		ListLoanCard Placeholder
+	<div class="list-loan-card row">
+		<div class="list-loan-card-desktop-column show-for-large large-4 xlarge-4 columns">
+			<div class="list-loan-card-desktop-column-image-wrapper">
+				<loan-card-image
```

Comment:
We need to structure the markup so that the loan card image component is not duplicated. Text copy can be duplicated where needed but not the images.